### ISSUE TITLE
Headset buttons not fully working in default music player.

### ISSUE THREAD

**Issue by raulillo82** [vegetalte-dev/android_device_bq_vegetalte#7]
I have a bluetooth headset. In previous versions I could use the button on it to play/pause the music. That does not work any more. There's also a wheel which is supposed to go to previous or next song, doesn't work either. Long press on the wheel will turn up/down the volume, this DOES work well.

Everything is working fine in vlc or google music players.
I'm using last week's build, the one released on March 6th or 7th, CM 13 (updating to newest one now)

**Comment by Kra1o5** [comment]
Hi @raulillo82 ,

Can you try this: https://github.com/downloads/chrisboyle/keytest/KeyTest.apk

& copy the output here?

**Comment by raulillo82** [comment]
Hi @Kra1o5 ,

Volume - (short press) -> Keycode: 88
KeyEvent: action=ACTION_DOWN, keyCode=KEYCODE_MEDIA_PREVIOUS, scanCode=165, metaState=0, flags=0x8, repeatCount=10, eventTime, downTime -> those two depend on when i press the key, deviceID=16, source=0x101

Volume + (short press) -> Keycode: 87
KeyEvent: the same than the other one, changing keyCode to KEYCODE_MEDIA_NEXT and scanCode=163

Play/pause: Keycode: 126
KeyEvent: the same than the other ones, changing keyCode to KEYCODE_MEDIA_PLAY and scanCode=200

Edit: Long press in volume - and volume +, which actually controls the volume, give the same output than the short press (which should control next/previous song).

**Comment by Kra1o5** [comment]
Hi @raulillo82 ,

Sorry for big delay.

It will be in next build.

**Comment by raulillo82** [comment]
Still doesn't work for me unfortunately :(
I deleted my bluetooth device and added it again. I deleted the cache for the music default application. Still no result, just volume controls work, start/stop button or next/previous song won't do anything.

Everything still works fine with vlc though. I'm using the build which was released yesterday (April 17th) and the commit seems to be included.
Any other advise will be welcome! Thanks.
